### PR TITLE
Сreated acceptCooperation modal

### PR DESCRIPTION
**Pull request by MickhaelSh**
image
image
image

### CODE REVIEW

File: src/components/status-chip/StatusChip.styles.ts [hunk 1]
```diff
@@ -5,13 +5,17 @@ import { StatusEnum } from '~/types'
 const statusColors = {
   pending: palette.basic.blue,
   active: palette.success[600],
-  closed: palette.primary[400]
+  closed: palette.primary[400],
+  'need action': palette.error[600]
```
Comment (Tolik170) [inline src/components/status-chip/StatusChip.styles.ts:9]: I think you can use created below enum here

File: src/components/user-profile-info/UserProfileInfo.styles.ts [hunk 2]
```diff
@@ -11,11 +13,18 @@ export const styles = {
...
-    width: '48px'
+    width: '48px',
+    '&:hover': {
+      transform: 'scale(1.1)'
+    }
   },
   name: {
     color: 'primary.500',
-    typography: 'button'
+    typography: 'button',
```
Comment (Tolik170) [inline src/components/user-profile-info/UserProfileInfo.styles.ts:23]: move "button" to enum

File: src/containers/my-cooperations/accept-cooperation-modal/AcceptCooperation.styles.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    alignItems: 'center'
+  },
+  form: {
+    minWidth: { xs: '280px', sm: '440px' },
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '36px',
+    alignItems: 'start'
+  },
+  title: { typography: 'h4' },
```
Comment (Tolik170) [inline src/containers/my-cooperations/accept-cooperation-modal/AcceptCooperation.styles.ts:16]: enum

File: src/containers/my-cooperations/accept-cooperation-modal/AcceptCooperationModal.tsx [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+          disabled={!isDirty}
+          type={ButtonTypeEnum.Submit}
+          variant={ButtonVariantEnum.ContainedLight}
+        >
+          {t('cooperationsPage.acceptModal.resend')}
+        </AppButton>
+      )}
+      <Box sx={styles.buttonRow}>
+        {needAction && (
+          <AppButton onClick={() => void handleAcceptCooperation()}>
```
Comment (Tolik170) [inline src/containers/my-cooperations/accept-cooperation-modal/AcceptCooperationModal.tsx:139]: better move this onClick outside

Comment (MickhaelSh) [reply src/containers/my-cooperations/accept-cooperation-modal/AcceptCooperationModal.tsx]: fixed

File: src/containers/my-cooperations/accept-cooperation-modal/AcceptCooperation.styles.ts (hunk 3 above)
Comment (MickhaelSh) [reply src/containers/my-cooperations/accept-cooperation-modal/AcceptCooperation.styles.ts]: fixed

File: src/components/user-profile-info/UserProfileInfo.styles.ts (hunk 2 above)
Comment (MickhaelSh) [reply src/components/user-profile-info/UserProfileInfo.styles.ts]: fixed

File: src/containers/guest-home-page/cards-with-button/CardsWithButton.tsx [hunk 5]
```diff
@@ -37,38 +39,38 @@ const CardsWithButton: FC<CardsWithButtonProps> = ({
...
+          sx={[
+            styles[boxSide].box,
+            state === 'exiting' && styles[boxSide].slidesIn,
+            state === 'entering' && styles[boxSide].slidesIn
+          ]}
+        >
+          <Box sx={styles[boxSide].clearBox} />
+          <Box sx={styles.image}>
+            <Box component='img' src={item.image} />
+            <Box className='dots' component='img' src={dots} />
```
Comment (Yurenko) [inline src/containers/guest-home-page/cards-with-button/CardsWithButton.tsx:58]: ComponentEnum

**Review by Yurenko — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**